Layout_style_v1.css

```diff
@@ -146,31 +145,79 @@ ul li:hover ul {
   grid-column-end: span 2;
   padding-bottom: 10px;
   height: 20px;
-  width: 2000px;
+  width: 1990px;
   background-color: #87CEEB;
   color: #FBCA01; 
 }
 
-.sidebar {
-	background-color: #6FC0DB;
-	height: 1500px;
-	width: 200px;
-	float: left;
+/* style for slider starts here */
+.slider {
+    position: relative;
+    width: 1990px;  /* adjustment to show each image on carousel. */
+    overflow: hidden;  /* overflow hides next image. */
+	background-color: #87CEEB;;
+	height: 450px;
+}
+.images {
+    display: flex;
+    width: 100%;
+}
+.images img {
+    height: 400px;
+    width: 100%;
+    transition: all 0.15s ease;  /* timing transition to show next image. */
+}
+.images input {
+    display: none;  /* display none hides the 3 images behind 1st image. */
+}
+.dots {
+    display: flex;
+    justify-content: center;
+    margin: 5px;
+}
+.dots label {
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+    border: solid #000000 3px;
+    cursor: pointer;
+    transition: all 0.15s ease;  /* timing transition to show next image. */
+    margin: 5px;
+}
+.dots label:hover {background: #000;}
+#img1:checked ~ .m1 {  /* ~ (tilde) is a selector used to select each "m" element following checked. */
+    margin-left: 0;
+}
+#img2:checked ~ .m2 {
+    margin-left: -100%;
+}
+#img3:checked ~ .m3 {
+    margin-left: -200%;  /* margin-left value moves each image to the Left. */
+}
+#img4:checked ~ .m4 {
+    margin-left: -300%;
 }
 
+/* style for slider ends here */
+
 .content{
 	box-sizing: border-box;
-	background-color: #FFFFFF;
+	background-color: #87CEEB;
 	position: relative;
-	width: 2000px;
-	height: 1500px;
-	left: 200px;
-	color: #000000;
+	width: 1990px;
+	height: 1300px;

```

Comment:
Changed the height of the content to 1300px as 1500px is too long. This is because I don't need that much space as there isn't that much text/images I need to add there.